src/saveSvgAsPng.js

```diff
@@ -270,22 +271,30 @@
         foreignObject.setAttributeNS(xmlNs, 'xmlns', foreignObject.tagName === 'svg' ? svgNs : xhtmlNs);
       });
 
-      return inlineCss(el, options).then(css => {
-        const style = document.createElement('style');
-        style.setAttribute('type', 'text/css');
-        style.innerHTML = `<![CDATA[\n${css}\n]]>`;
-
-        const defs = document.createElement('defs');
-        defs.appendChild(style);
-        clone.insertBefore(defs, clone.firstChild);
-
-        const outer = document.createElement('div');
+      if (excludeCss) {
+        var outer = document.createElement('div');
         outer.appendChild(clone);
-        const src = outer.innerHTML.replace(/NS\d+:href/gi, 'xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href');
-
+        var src = outer.innerHTML;
```

Comment:
These two `var`s can be `const`, correct? If so, `const` is preferred.

Reply:
Yes, definitely. Should be fixed now.